ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/mgmt_interface_description.yml

```diff
@@ -8,4 +8,5 @@ l2leaf:
   nodes:
     - name: mgmt_interface_description
       mgmt_ip: 1.1.1.2
```

Comment:
so in the other example `mgmt_ip` has mask like `192.168.201.201/24` can you please use this PR as an oppotrunity to fix this:

```
mgmt_ip: 1.1.1.2/24
```

Reply:
Updated